internal/models/handler/match.go

```diff
@@ -0,0 +1,25 @@
+package handlermodel
+
+import (
+	"time"
+)
```

Comment:
Lets delete folder handler and repo and just use different names for structures

Reply:
why?

Reply:
so that it was in one package corresponding to models folder. renaming packages is not so good idea

Reply:
alright, but if one day I'll get confused looking for specific file it will be on you ;)

Reply:
done, resolve it if you are satisfied

Reply:
You got me wrong, I mean put all user structures(userDB user Handler) to the files user.go